components/profile/pages/profile-page/view.tsx

```diff
@@ -59,6 +66,9 @@ export default function ProfilePageView({
   onSave,
   onResend,
   onSwitchChange,
+  hideConnectModal,
+  handleChangeMyhandle,
```

Comment:
rename all "action" function to have a "on" as in, "onChangeMyHandleClick", "onHideModalClick" -- that way people will understand if they should send a prop or not _and_ will better understand _what it actually is_